test/e2e-tests.sh

```diff
@@ -54,8 +54,6 @@ fi
 
 if (( HTTPS )); then
   use_https="--https"
-  # TODO: parallel 1 is necessary until https://github.com/knative/serving/issues/7406 is solved.
```

Comment:
do we still need it in MESH above?

Reply:
more importantly, what's keeping it there?

Reply:
For mesh mode, it had this problem https://github.com/knative/serving/pull/5989 long time ago. In short, ksvc took long time to become ready in `TestScaleToN/scale-100` due to CPU high load. 

I personally think that MESH should keep `parallel 1` option. I am removing `parallel 1` from https because the test grid is [green](https://testgrid.knative.dev/serving#https) with the option. But MESH is [not green](https://testgrid.knative.dev/serving#istio-stable-mesh) even though it has `parallel 1` option. MESH should fix the issue with the option first.

Reply:
Got it, thanks for the explanation.